common/src/main/java/com/wynntils/models/players/PartyModel.java

```diff
@@ -78,7 +78,8 @@ public final class PartyModel extends Model {
 
     private static final Pattern PARTY_CREATE_SELF = Pattern.compile("§eYou have successfully created a party\\.");
 
-    private static final Pattern PARTY_INVITED = Pattern.compile("§eYou have been invited to join (.*)'s? party\\.");
+    private static final Pattern PARTY_INVITED =
+            Pattern.compile("\\s+§eYou have been invited to join (\\w{1,16})'s party!");
```

Comment:
Please keep the `?` after the `s`, I'm pretty sure that still is valid for user names ending in s.

Reply:
Oh yeah I didn't even notice I changed this. My bad